packages/batch-submitter/src/utils.ts

```diff
@@ -21,3 +21,10 @@ export const remove0x = (str: string): string => {
     return str
   }
 }
+
+export const formatNumber = (num: any): number => {
+  if (num === '0x') {
+    return 0
+  }
+  return BigNumber.from(num).toNumber()
+}
```

Comment:
i believe i deprecated the entire `utils` file in batch submitter while adding logs, since everything is either duplicated in core-utils or unused: https://github.com/ethereum-optimism/batch-submitter/pull/73/commits/0c05ec7894ccf196e83ab8e74188d95205e11d24

Reply:
Removed the raw call! Note that we weren't exposing `rawTransaction` before so I added that to the l2 context